GW2EIEvtcParser/EIData/DamageModifiers/CommonDamageModifiers/ItemDamageModifiers.cs

```diff
@@ -25,6 +25,21 @@ internal static class ItemDamageModifiers
         internal static readonly List<DamageModifierDescriptor> IncomingDamageModifiers = new List<DamageModifierDescriptor> {
             new BuffOnActorDamageModifier(FractalDefensive, "Fractal Defensive", "5% per stack", DamageSource.NoPets, 5.0, DamageType.StrikeAndCondition, DamageType.All, Source.Item, ByStack, BuffImages.FractalDefensive, DamageModifierMode.PvE),
             // Regroup consumables that reduce inc damage by their reduction instead of adding one modifier per consumable to reduce cluttering
+            new BuffOnActorDamageModifier(CanOfStewedOysters, "Food: -8%", "-8%", DamageSource.All, -8.0, DamageType.Strike, DamageType.All, Source.Item, ByPresence, BuffImages.NourishmentFood, DamageModifierMode.PvEWvW),
+            new BuffOnActorDamageModifier(
+                new long[]
+                {
+                    PlateOfPeppercornSpicedCoqAuVin, MistsInfusedPeppercornCrustedSousVideSteak, PeppercornCrustedSousVideSteak, PlateOfPepperedClearTruffleRavioli,
+                    PlateOfPeppercornSpicedBeefCarpaccio, PlateOfPeppercornSpicedPoultryAspic, PeppercornSpicedEggsBenedict, SpicedPepperCremeBrulee,
+                    SpicedPeppercornCheesecake, PepperedCuredMeatFlatbread, PeppercornAndVeggieFlatbread, BowlOfSpicedFruitSalad, 
+                    SpherifiedPeppercornSpicedOysterSoup, MistsInfusedSpherifiedPeppercornSpicedOysterSoup,
+                }, "Ascended Food: -10%", "-10%", DamageSource.NoPets, -10.0, DamageType.Strike, DamageType.All, Source.Item, ByMultiPresence, BuffImages.NourishmentFood, DamageModifierMode.PvEWvW),
```

Comment:
ByPresence, although impossible, ByMultiPresence will stack the bonus per present nourishment on the player